src/main/java/io/kuzzle/sdk/listeners/OnQueryDoneListener.java

```diff
@@ -2,7 +2,13 @@
 
 import org.json.JSONObject;
 
-public interface OnQueryDoneListener {
-  void onSuccess(JSONObject response);
-  void onError(JSONObject error);
+public abstract class OnQueryDoneListener {
+  public abstract void onSuccess(JSONObject response);
+  public void onError(JSONObject error) {
+    if (error != null) {
+      System.err.println("Default error handler invoked: " + error.toString());
```

Comment:
Despite the fact I think it's a good idea for  onError to stay abstract and force the implementation of it, because we are on android system we should not use System.err but Log.e(TAG, MESSAGE);